### PR TITLE
Lazy load locales

### LINKED ISSUE #1345: Lazy load locales
### Clear and concise description of the problem

Currently all locales are loaded immediately. This unnecessarily bloats the bundle size.

### Suggested solution

Lazy load locales: https://vue-i18n.intlify.dev/guide/advanced/lazy.html

### Alternative

_No response_

### Additional context

_No response_

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by lukashass**
Closes #1345 

`index.js` size reduction: `538.71 KiB` -> `382.74 KiB`

Also sets correct html `lang` attribute.

## TODO

- [x] check for alternatives for vite plugin to load supported locales https://github.com/intlify/bundle-tools/issues/205

### CODE REVIEW

File: web/src/views/RepoAdd.vue [hunk 1]
```diff
@@ -78,7 +78,7 @@ export default defineComponent({
     const { doSubmit: activateRepo, isLoading: isActivatingRepo } = useAsyncAction(async (repo: Repo) => {
       repoToActivate.value = repo;
       await apiClient.activateRepo(repo.owner, repo.name);
-      notifications.notify({ title: i18n.t('repo.enabled.success'), type: 'success' });
+      notifications.notify({ title: i18n.t('repo.enable.success'), type: 'success' });
```
Comment (6543) [inline web/src/views/RepoAdd.vue:81]: nice catch!

**Comment by 6543** [comment]
to test if key is right on linux via cli:
```sh
cat web/src/assets/locales/en.json | jq .repo.enable.success
```
if you get your value the key is right ;)

**Comment by lukashass** [comment]
I couldn't get lazy loading of dayjs and TimeAgo locales to work, so I would suggest to try that later.

**Review by lafriks — APPROVED** [review]

**Comment by 6543** [comment]
Thanks :)

**Pull request merged**